app/static/js/libs/marked.js

```diff
@@ -841,6 +841,19 @@ Renderer.prototype.listitem = function(text, checked) {
     + '</li>\n';
 };
 
+Renderer.prototype.listitemTODO = function(text, checked) {
```

Comment:
I think it's better. But this is not what I originally mean. Do you know jQuery? If we want to extend jQuery, we usually don't modify the source code of jQuery. It exposes an interface, we can extend our function on that interface without changing jQuery source code. marked.js is also a popular library, it provides some API to extend its functionality. But now we changed its source code, maybe it's not easy to integrate task items code with marked.js.